src/lib.rs

```diff
@@ -1708,44 +1719,28 @@ fn encode_frame(sequence: &mut Sequence, fi: &mut FrameInvariants, fs: &mut Fram
             None => (),
         }
     } else {
-        let obu_extension = 0;
+        let tile = encode_tile(fi, fs); // actually tile group
+        let obu_payload_size = tile.len() as u64;
         let mut buf1 = Vec::new();
         {
             let mut bw1 = BitWriter::<BE>::new(&mut buf1);
-            bw1.write_obu_header(OBU_Type::OBU_FRAME, obu_extension).unwrap();
```

Comment:
So, in Feb version, encoder does not write OBU_FRAME if there is one tile ony?
I guessed decoder can still decode even though OBU_FRAME_HEADER comes in (the switch..case flow is written that way), but if it is OBU_FRAME_HEADER, i.e "if (obu_header.type != OBU_FRAME) break;"  (or show_existing_frame == true,) then the obu decoding does not continue to OBU_TILE_GROUP case which actually decode a tile, i.e. inside aom_decode_frame_from_obus(), in current version of libaom I mean.

Reply:
OK, then we'll need to patch this to support OBU_FRAME once that's added.